R-proj/src/sample_points.cpp

```diff
@@ -351,13 +367,18 @@ Rcpp::NumericMatrix sample_points(Rcpp::Nullable<Rcpp::Reference> P,
     NT a = 0.5;
     if (Rcpp::as<Rcpp::List>(distribution).containsElementNamed("variance")) {
         a = 1.0 / (2.0 * Rcpp::as<NT>(Rcpp::as<Rcpp::List>(distribution)["variance"]));
-        if (!gaussian) {
+        if (walk == exponential_hmc) a = Rcpp::as<NT>(Rcpp::as<Rcpp::List>(distribution)["variance"]);
```

Comment:
why `a` and not `variance`?

Reply:
Because in the Gaussian case we set a = 1/2*variance (it is more simple to handle it with the calculations).
I chose not to declare a new variable for the exponential dist.